chapters/synchronization.txt

```diff
@@ -4236,6 +4252,52 @@ ifdef::VK_NV_mesh_shader[]
     pname:dstStageMask must: not contain
     ename:VK_PIPELINE_STAGE_TASK_SHADER_BIT_NV
 endif::VK_NV_mesh_shader[]
+  * Each element of pname:pMemoryBarriers must: not have any access flag
+    included in its pname:srcAccessMask member that is not supported by any
+    of the pipeline stages in pname:srcStageMask, as specified in the
+    <<synchronization-access-types-supported, table of supported access
+    types>>
+  * Each element of pname:pMemoryBarriers must: not have any access flag
+    included in its pname:dstAccessMask member that is not supported by any
+    of the pipeline stages in pname:dstStageMask, as specified in the
+    <<synchronization-access-types-supported, table of supported access
+    types>>
+  * For each element of pname:pBufferMemoryBarriers, if its
+    pname:srcQueueFamilyIndex and pname:dstQueueFamilyIndex
+    members describe a <<synchronization-queue-transfers-release, queue
+    family release operation>>, and its pname:buffer member was created
+    with a sharing mode of ename:VK_SHARING_MODE_EXCLUSIVE, then its
+    pname:dstAccessMask member must: only contain access flags that are
+    supported by any of the pipeline stages in pname:dstStageMask, as
+    specified in the <<synchronization-access-types-supported, table of
+    supported access types>>
+  * For each element of pname:pBufferMemoryBarriers, if its
+    pname:srcQueueFamilyIndex and pname:dstQueueFamilyIndex
+    members describe a <<synchronization-queue-transfers-acquire, queue
```

Comment:
I belive it should be opposite. "If it does **not** describe a queue family acquire operation".

Reply:
Whoops